heron/statemgrs/src/java/com/twitter/heron/statemgr/zookeeper/curator/CuratorStateManager.java

```diff
@@ -208,6 +209,10 @@ public String getConnectionString() {
       LOG.info("Deleted node for path: " + path);
       result.set(true);
 
+    } catch (KeeperException.NoNodeException e) {
+      LOG.info(path + " does not exist so there's nothing to delete");
```

Comment:
I think LOG.warn would be justified, since code shouldn't be calling delete on something that they don't expect to be set.